src/main/java/org/apache/maven/plugins/javadoc/JavadocUtil.java

```diff
@@ -118,32 +119,54 @@ public class JavadocUtil {
                     + "environment variable using -Xms:<size> and -Xmx:<size>.";
 
     /**
-     * Method that removes the invalid directories in the specified directories. <b>Note</b>: All elements in
-     * <code>dirs</code> could be an absolute or relative against the project's base directory <code>String</code> path.
+     * Method that removes invalid classpath elements in the specified paths. <b>Note</b>: All elements in
+     * <code>paths</code> could be absolute or relative against the project's base directory <code>String</code> path.
+     * When pruning class paths, you can optionally include "*.jar" files in the result, otherwise only directories are
+     * permitted.
      *
      * @param project the current Maven project not null
-     * @param dirs the collection of <code>String</code> directories path that will be validated.
-     * @return a List of valid <code>String</code> directories absolute paths.
+     * @param paths the collection of <code>String</code> paths that will be validated
```

Comment:
the collection of <code>String</code> paths --> the paths
Javadoc will insert the types

Reply:
Why are you asking questions about existing comments in your product, which I have just copied/moved and slightly adjusted? If it was good enough so far, it should be good enough now.

Reply:
Because you copied/moved and slightly adjusted them. You also added new emthods that weren't here before.

Reply:
You don't get it, do you? Then have your way, I could not care less. If you cannot see that this is PR is an improvement and refuse to accept it, you are just shooting in your own foot.